### PR TITLE
DOC-4389: Add legacy social intranet, the stream and widgets in the D…

### PR DESCRIPTION
**Pull request by hghorbel**
…W documentation

### CODE REVIEW

**Review by plamarque — CHANGES_REQUESTED** [review]
please change images with more meaningfull usernames than abcde199

File: docs/GettingStarted.rst [hunk 1]
```diff
@@ -303,17 +308,22 @@ redirected to the :ref:`Social Intranet homepage <PLFHomepage>`. Now, you can st
...
 
-==========================
-Digital Workplace Homepage
-==========================
+==============================
+Navigate in Digital Workplace
+==============================
 
-This section introduces you to the Digital Workplace homepage, ie **Snapshot**. Besides,
+This section introduces you to the navigation in the Digital Workplace, ie **Snapshot**. Besides,
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
This section introduces the navigation in the Digital Workplace site. Besides,
```

File: docs/GettingStarted.rst [hunk 2]
```diff
@@ -303,17 +308,22 @@ redirected to the :ref:`Social Intranet homepage <PLFHomepage>`. Now, you can st
...
-   Introduction to applications that come with the Digital Workplace
-   homepage.
-   
+ * :ref:`Navigation <Navigation>`
+   How to navigate in the Digital Workplace.
+
+ * :ref:`Snapshot <Snapshot>`
+   Introduction to the Digital Workplace Homepage.
+
+ * :ref:`Digital Workplace applications <DW-Gadgets>`
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
 * :ref:`Digital Workplace widgets <DW-Gadgets>`
```

File: docs/GettingStarted.rst [hunk 3]
```diff
@@ -303,17 +308,22 @@ redirected to the :ref:`Social Intranet homepage <PLFHomepage>`. Now, you can st
...
+   Introduction to applications that come with the Digital Workplace.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
   Introduction to widget blocks that come with the Digital Workplace.
```

File: docs/GettingStarted.rst [hunk 4]
```diff
@@ -442,155 +452,16 @@ Quickly perform key actions through the following applications:
...
-3. Click Next to be redirected to the Wiki application of your selected
-   location. Here, you can :ref:`navigate across spaces <Navigate-across-spaces>`.
-
-4. Enter the content of your Wiki page. See :ref:`Creating a page <Creating-New-Page>`
-   for more details.
-
-.. _Intranet-Gadgets:
-
-Social Intranet applications
+Digital Workplace applications
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
Digital Workplace widgets
```

File: docs/GettingStarted.rst [hunk 5]
```diff
@@ -442,155 +452,16 @@ Quickly perform key actions through the following applications:
...
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Intranet applications are ones which come with the Social Intranet
-homepage, including:
+Digital Workplace applications are ones which come with the Digital Workplace, including:
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
Digital Workplace widgets come with the Digital Workplace, including:
```

File: docs/GettingStarted.rst [hunk 6]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
 
 After each action is performed, it will be remarked as completed with a
-strike-through even though it is not performed via this application.
-Also, the completion percentage is updated on the percentage bar.
+check-mark even though it is not performed via this application.
 
-When all the actions are performed, the completion percentage will be
-100%. You can remove this application from the homepage by clicking
-Close or by hovering your cursor over the application header, and click |image44|.
+When all the actions are performed, you will can remove this application from the Stream Page by clicking
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
When all the actions are performed, you can remove this widget from the Stream Page by clicking
```

File: docs/GettingStarted.rst [hunk 7]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
+Close on the application header|DW-2|.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
Close on the header|DW-2|.
```

File: docs/GettingStarted.rst [hunk 8]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
 
-|image45|
+    .. note:: - You cannot get the Getting Started application back when it is removed.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
    .. note:: - You can’t get the Getting Started widget back when it is removed.
```

File: docs/GettingStarted.rst [hunk 9]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
----------
+Welcome Back
+-------------
 
-The Calendar application displays some calendars and all of their events
-and tasks scheduled in the Calendar applications of Intranet and spaces.
-When going to the homepage, you will see events with their start and end
-date and tasks of Today. You can also see the events and tasks of the
-previous/next day by clicking the previous/next arrow respectively.
+The welcome back widget is an application that welcomes you with some information about your spaces, connections,
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
The welcome back widget is an block that welcomes you with some information about your spaces, connections, 
```

File: docs/GettingStarted.rst [hunk 10]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
+Welcome Back
+-------------
 
-The Calendar application displays some calendars and all of their events
-and tasks scheduled in the Calendar applications of Intranet and spaces.
-When going to the homepage, you will see events with their start and end
-date and tasks of Today. You can also see the events and tasks of the
-previous/next day by clicking the previous/next arrow respectively.
+The welcome back widget is an application that welcomes you with some information about your spaces, connections, 
+your points obtained in the current week and your weekly rank.
```
Comment (plamarque) [inline docs/GettingStarted.rst]: rank is probably not clear at this stage to a new comer. We should link to the section in gamification where rank concept is described.

File: docs/GettingStarted.rst [hunk 11]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
+|DW-3|
 
--  To view details of an event/task directly in the Calendar
-   application, click your desired event/task.
+- You will see your avatar and your name just after the welcome Back label.
 
--  To configure and set which calendars to be displayed in the Calendar
-   application, hover your cursor over the application, then click
-   |image47| at the right bottom of the application.
+- For your spaces, you will see the total number of spaces that you're member of, or created.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
- For your spaces, you will see the total number of spaces that you're member of.
```

File: docs/GettingStarted.rst [hunk 12]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
+  If you have some requests to join a space, you will find a badge displaying the number of requests
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
  If you have some invitations to join spaces, you will find a badge displaying the number of awaiting invitations
```

File: docs/GettingStarted.rst [hunk 13]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
+  you have and you can just click on them to display them in a drawer.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
  you have and you can just click on it to display them in a drawer.
```

File: docs/GettingStarted.rst [hunk 14]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
 
-   -  To add one of removed calendars again to the list of Displayed
-      Calendars, simply hover your cursor over the desired calendar,
-      then click |image48|. You can use the Search box to filter
-      calendars quickly.
+    .. note:: - If you don't have any request to join a space, the badge will be hidden.
 
-   -  Click **OK** to accept your settings.
+- You will have your Weekly gamifications gained points and by clicking on your points, you will have a pie chart
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
- You can see the points you earned his week and by clicking on your points, you will have a pie chart 
```

File: docs/GettingStarted.rst [hunk 15]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
 
-   -  To add one of removed calendars again to the list of Displayed
-      Calendars, simply hover your cursor over the desired calendar,
-      then click |image48|. You can use the Search box to filter
-      calendars quickly.
+    .. note:: - If you don't have any request to join a space, the badge will be hidden.
 
-   -  Click **OK** to accept your settings.
+- You will have your Weekly gamifications gained points and by clicking on your points, you will have a pie chart 
+  that presents your points partition by domains.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
  that presents your points distribution by domain.
```

File: docs/GettingStarted.rst [hunk 16]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
 
-    .. note:: When a task is completed, it will be remarked with a strike-through.
+|DW-4|
 
-.. _InvitationsApp:
+- You will have also your Weekly rank and by clicking on your rank, you will have a leaderboad
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
- You can also your rank for this week and by clicking on your rank, you can see a leaderboard 
```

File: docs/GettingStarted.rst [hunk 17]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
-   -  Click **OK** to accept your settings.
+- You will have your Weekly gamifications gained points and by clicking on your points, you will have a pie chart 
+  that presents your points partition by domains.
 
-    .. note:: When a task is completed, it will be remarked with a strike-through.
+|DW-4|
 
-.. _InvitationsApp:
+- You will have also your Weekly rank and by clicking on your rank, you will have a leaderboad 
+  that presents your position comparing to you collaborators and your weekly gained points.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
  that presents your position comparing to other members and the amount of points you earned this week so far.
```

File: docs/GettingStarted.rst [hunk 18]
```diff
@@ -753,6 +589,25 @@ corresponding button at the pop-up bottom:
 
     .. note:: From the pop-up, you can click his avatar or display name to jump to his activity stream page.
 
+.. _Perkstore:
+
+Perkstore
+----------
+
+The My Orders widget will presents to you the number of your current orders in the Perkstore.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
The Perks Store widget will presents to you the number of your current orders in the Perkstore.
```

File: docs/GettingStarted.rst [hunk 19]
```diff
@@ -753,6 +589,25 @@ corresponding button at the pop-up bottom:
 
     .. note:: From the pop-up, you can click his avatar or display name to jump to his activity stream page.
 
+.. _Perkstore:
+
+Perkstore
```
Comment (plamarque) [inline docs/GettingStarted.rst]: Perks Store

File: docs/GettingStarted.rst [hunk 20]
```diff
@@ -753,6 +589,25 @@ corresponding button at the pop-up bottom:
...
+----------
+
+The My Orders widget will presents to you the number of your current orders in the Perkstore.
+By clicking on this number, the Perkstore page will be opened on the list of all your orders.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
By clicking on this number, the Perks Store application will be opened on the list of all your orders.
```

File: docs/GettingStarted.rst [hunk 21]
```diff
@@ -753,6 +589,25 @@ corresponding button at the pop-up bottom:
...
+
+|DW-8|
+
+.. _Wallet:
+
+Wallet Balance
+---------------
+
+The Wallet widget will presents to you the number of your cauris on your wallet.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
The Wallet widget displays the number of your balance of your wallet.
```

File: docs/GettingStarted.rst [hunk 22]
```diff
@@ -753,6 +589,25 @@ corresponding button at the pop-up bottom:
...
+Cauris are earned by the points you gained on gamification when you react and collabore with your collaborators.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
You will receive tokens regularly in your wallet by collaborating through the digital workplace apps. 
```
Should link to the section of the doc that talks about rewards in wallet

**Review by plamarque — CHANGES_REQUESTED** [review]
reviewed text and suggested many language changes.
Generally speaking avoid using future tense for describing interfaces. Present is better.

File: docs/GettingStarted.rst [hunk 23]
```diff
@@ -14,15 +14,20 @@ This chapter covers the following topics:
 
        Introduction to what you need to do for the first startup of
        eXo Platform.
+
+:ref:`Navigate on the Digital Workplacce <DWHomepage>`
+
+       Introduction to the navigation on the Digital Workplace, the Digital Workplace
+       applications, Spaces and Settings.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
       widgets, spaces and settings.
```

File: docs/GettingStarted.rst [hunk 24]
```diff
@@ -303,17 +308,22 @@ redirected to the :ref:`Social Intranet homepage <PLFHomepage>`. Now, you can st
...
    
- * :ref:`Digital Workplace applications <Intranet-Gadgets>`
-   Introduction to applications that come with the Digital Workplace
-   homepage.
-   
+ * :ref:`Navigation <Navigation>`
+   How to navigate in the Digital Workplace.
+
+ * :ref:`Snapshot <Snapshot>`
+   Introduction to the Digital Workplace Homepage.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
   Introduction to the Snapshot page, all your Digital Workplace at a glance .
```

File: docs/GettingStarted.rst [hunk 25]
```diff
@@ -442,155 +452,16 @@ Quickly perform key actions through the following applications:
...
-.. _Creating a Wiki: 
+*This section will be developed soon*
 
-Creating a Wiki page
----------------------
-
-1. Click Wiki Page from the drop-down menu.
-
-|image42|
+.. _DW-Gadgets:
```
Comment (plamarque) [inline docs/GettingStarted.rst]: Widgets

File: docs/GettingStarted.rst [hunk 26]
```diff
@@ -442,155 +452,16 @@ Quickly perform key actions through the following applications:
...
-
-.. _Intranet-Gadgets:
-
-Social Intranet applications
+Digital Workplace widgets
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Intranet applications are ones which come with the Social Intranet
-homepage, including:
+Digital Workplace widgets come with the Digital Workplace, including:
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
Digital Workplace comes with several widget applications preinstalled on default pages to provide useful info and handy shortcuts to all apps and key information such as :
```

File: docs/GettingStarted.rst [hunk 27]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
 
 After each action is performed, it will be remarked as completed with a
-strike-through even though it is not performed via this application.
-Also, the completion percentage is updated on the percentage bar.
+check-mark even though it is not performed via this application.
 
-When all the actions are performed, the completion percentage will be
-100%. You can remove this application from the homepage by clicking
-Close or by hovering your cursor over the application header, and click |image44|.
+When all the actions are performed, you can remove this widget from the Stream Page by clicking
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
When all the actions are performed, you can dismiss this widget permanently by clicking
```

File: docs/GettingStarted.rst [hunk 28]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
+Close on the header|DW-2|.
 
-|image45|
+    .. note:: - You can’t get the Getting Started widget back when it is removed.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
    .. note:: - You can’t get the Getting Started widget back after you closed it.
```

File: docs/GettingStarted.rst [hunk 29]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
+Welcome Back
+-------------
 
-The Calendar application displays some calendars and all of their events
-and tasks scheduled in the Calendar applications of Intranet and spaces.
-When going to the homepage, you will see events with their start and end
-date and tasks of Today. You can also see the events and tasks of the
-previous/next day by clicking the previous/next arrow respectively.
+The welcome back widget is an block that welcomes you with some information about your spaces, connections, 
+your points obtained in the current week and your weekly rank.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
the points you earned and the rank you reached this week.
```

File: docs/GettingStarted.rst [hunk 30]
```diff
@@ -753,6 +589,25 @@ corresponding button at the pop-up bottom:
 
     .. note:: From the pop-up, you can click his avatar or display name to jump to his activity stream page.
 
+.. _Perkstore:
+
+Perkstore
+----------
+
+The Perks Store widget will presents to you the number of your current orders in the Perkstore.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
The Perks Store widget displays pending orders you have in the Perks Store application.
```

should add a link to Pers Store doculentztion section

File: docs/GettingStarted.rst [hunk 31]
```diff
@@ -753,6 +589,25 @@ corresponding button at the pop-up bottom:
...
+By clicking on this number, the Perks Store application will be opened on the list of all your orders.
+
+|DW-8|
+
+.. _Wallet:
+
+Wallet Balance
+---------------
+
+The Wallet widget displays the number of your balance of your wallet.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
The Wallet widget displays the token balance of your wallet.
```

File: docs/GettingStarted.rst [hunk 32]
```diff
@@ -753,6 +589,25 @@ corresponding button at the pop-up bottom:
...
+You will receive tokens regularly in your wallet by collaborating through the digital workplace apps.
```
Comment (plamarque) [inline docs/GettingStarted.rst:610]: add a see Rewards (link to Reward sextion in the doc)

File: docs/GettingStarted.rst [hunk 33]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
-previous/next day by clicking the previous/next arrow respectively.
+The welcome back widget is an block that welcomes you with some information about your spaces, connections, 
+your points obtained in the current week and your weekly rank.
 
-|image46|
+|DW-3|
 
--  To view details of an event/task directly in the Calendar
-   application, click your desired event/task.
+- You will see your avatar and your name just after the welcome Back label.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
- You can see your profile picture and your name just after the welcome Back label.
```

I hate the term avatar
turn this sentence to present tense. Future makes no sense when describing an interface.

File: docs/GettingStarted.rst [hunk 34]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
 
--  To configure and set which calendars to be displayed in the Calendar
-   application, hover your cursor over the application, then click
-   |image47| at the right bottom of the application.
+- For your spaces, you will see the total number of spaces that you're member of.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
- The widget also displays the  total number of spaces that you're member of.
```

File: docs/GettingStarted.rst [hunk 35]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
+  If you have some invitations to join spaces, you will find a badge displaying the number of awaiting invitations
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
  A counter of your pending space invitations is displayed in a small badge
```

File: docs/GettingStarted.rst [hunk 36]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
+  you have and you can just click on it to display them in a drawer.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
  Click the badge to display all invitations and easily accept or refuse them.
```

File: docs/GettingStarted.rst [hunk 37]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
 
-   -  To remove a calendar from the list of Displayed Calendars, click |image47|.
+    .. note:: - If you don't have any request to join a space, the badge will be hidden.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
    .. note:: - When you don't have any space invitation, the badge is hidden.
```

File: docs/GettingStarted.rst [hunk 38]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
 
-      This removed calendar will appear in the list of Display
-      Additional Calendar.
+- For your connections, you will see the total number of connections on your network.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
- this widget also reminds you the size of your social network by displaying your current number of Connections.
```

File: docs/GettingStarted.rst [hunk 39]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
+  If you have some connection requests, you will find a badge displaying the number of requests
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
  When you have some pending  connection requests, a small badge displays a requests counter.
```

File: docs/GettingStarted.rst [hunk 40]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
+  you have and you can just click on them to display them in a drawer.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
 Click the badge to display the list of pending requests and accept or refuse them.
```

File: docs/GettingStarted.rst [hunk 41]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
 
-   -  To add one of removed calendars again to the list of Displayed
-      Calendars, simply hover your cursor over the desired calendar,
-      then click |image48|. You can use the Search box to filter
-      calendars quickly.
+    .. note:: - If you don't have any request to join a space, the badge will be hidden.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
    .. note:: - When you don’t have any connection request, the badge is hidden.
```

File: docs/GettingStarted.rst [hunk 42]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
 
-   -  Click **OK** to accept your settings.
+- You can see the points you earned his week and by clicking on your points, you will have a pie chart
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
- You can see the points you earned this week and by clicking on your points, you can see a pie chart 
```

File: docs/GettingStarted.rst [hunk 43]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
 
-   -  Click **OK** to accept your settings.
+- You can see the points you earned his week and by clicking on your points, you will have a pie chart 
+  that presents your points distribution by domain.
 
-    .. note:: When a task is completed, it will be remarked with a strike-through.
+|DW-4|
 
-.. _InvitationsApp:
+- You can also your rank for this week and by clicking on your rank, you can see a leaderboard
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
- You can also see your rank for this week and by clicking on your rank, you can see a leaderboard 
```

File: docs/GettingStarted.rst [hunk 44]
```diff
@@ -608,92 +479,59 @@ Intranet homepage applications on the top right. This application helps
...
-   -  Click **OK** to accept your settings.
+- You can see the points you earned his week and by clicking on your points, you will have a pie chart 
+  that presents your points distribution by domain.
 
-    .. note:: When a task is completed, it will be remarked with a strike-through.
+|DW-4|
 
-.. _InvitationsApp:
+- You can also your rank for this week and by clicking on your rank, you can see a leaderboard 
+  that presents your position comparing to other members and the amount of points you earned this week so far.
```
Comment (plamarque) [inline docs/GettingStarted.rst]:
```suggestion
  that presents your position comparing to other members and how many points you earned this week so far.
```

**Review by plamarque — CHANGES_REQUESTED** [review]
proofreading pass #2 to fix some typos inteoduced by pass #1 and suggest more language simplifications

File: docs/GettingStarted.rst (hunk 32 above)
Comment (hghorbel) [reply docs/GettingStarted.rst:610]: Done.

File: docs/GettingStarted.rst (hunk 30 above)
Comment (hghorbel) [reply docs/GettingStarted.rst]: We don't have documentation for perks store, all what i find is a link to github page for Perks sotre.

File: docs/GettingStarted.rst (hunk 10 above)
Comment (hghorbel) [reply docs/GettingStarted.rst]: Done.

**Comment by hghorbel** [comment]
@plamarque changes done

**Review by plamarque — APPROVED** [review]
LGTM

**Pull request merged**